Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/msg/IMessageRegistry.java

```diff
@@ -0,0 +1,13 @@
+package edu.asu.conceptpower.rest.msg;
+
+public interface IMessageRegistry {
+
+    /**
+     * This method returns the message converter object based on the media type.
+     * 
+     * @param mediaType
+     * @return
+     */
+    public IMessageConverter getMessageFactory(String mediaType);
```

Comment:
I have explained the valid values for mediatype are accept headers of XML and JSON.